ebs/ebs.go

```diff
@@ -423,7 +434,7 @@ func (d *Driver) DeleteVolume(req Request) error {
 		log.Debugf("Detached %v(%v) from %v", id, volume.EBSID, volume.Device)
 	}
 
-	if !referenceOnly {
+	if !referenceOnly && !volume.PreExist {
```

Comment:
um... Why? Normally we don't remove EBS volume unless user asked to remove it using Convoy CLI. Any call passed by plugin interface(Docker or K8s) shouldn't remove volume by default.